src/test/java/io/skai/okta/internshipstreamingserviceparserxml/repository/EpisodeAdditionalInfoRepositoryTest.java

```diff
@@ -0,0 +1,60 @@
+package io.skai.okta.internshipstreamingserviceparserxml.repository;
+
+import io.skai.okta.internshipstreamingserviceparserxml.dto.EpisodeAdditionalInfo;
+import io.skai.okta.internshipstreamingserviceparserxml.repository.impl.EpisodeAdditionalInfoRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@SpringBootTest
+public class EpisodeAdditionalInfoRepositoryTest {
+
+    @Autowired
+    private EpisodeAdditionalInfoRepository repository;
+
+    private EpisodeAdditionalInfo expectedInfo;
+
+    @BeforeEach
+    void setUp() {
+        expectedInfo = EpisodeAdditionalInfo.builder()
```

Comment:
you can init it in fields section, you don't have any dynamic info, there is no point to create same object again and again before each test

Reply:
if i used the same object can i put it in @BeforeAll?